lawnchair/res/values-de/strings.xml

```diff
@@ -324,37 +337,39 @@
     <string name="selected_count">%1$d ausgewählt</string>
     <string name="tab_manage_apps">Apps verwalten</string>
     <string name="tab_bottom_sheet_save">Speichern</string>
-    <string name="tab_hide_from_main">Verstecke App im Drawer</string>
-    <string name="tab_color">Farbe des Tabs</string>
-    <string name="none">Keine</string>
-    <string name="undo">Rückgängig machen</string>
+    <string name="tab_hide_from_main">In App-Liste verstecken</string>
+    <string name="tab_color">Tab-Farbe</string>
+    <string name="delete">Löschen</string>
+    <string name="none">Nichts</string>
+    <string name="undo">Rückgängig</string>
+    <string name="name">Name</string>
     <!--Marker J-->
     <string name="shortcut_sesame">Verknüpfungen</string>
     <string name="pref_integrations">Integrationen</string>
     <string name="pref_integrations_summary">Suche, Sesame, Google</string>
     <string name="sesame">Sesame</string>
     <string name="sesame_data_integration">Datenintegration</string>
-    <string name="sesame_upsell">Tippen um Sesame zu installieren</string>
+    <string name="sesame_upsell">Tippen, um Sesame zu installieren</string>
     <string name="sesame_search_upsell_on">Sesame ist als Suchanbieter festgelegt</string>
     <string name="sesame_search_upsell_off">Sesame ist nicht als Suchanbieter festgelegt</string>
-    <string name="pref_sesame_enabled">Sesame shortcut Daten geteilt</string>
+    <string name="pref_sesame_enabled">Daten für Sesame aktiviert</string>
     <string name="pref_sesame_disabled">Keine Daten geteilt</string>
     <string name="pref_sesame_show_shortcuts">App-Verknüpfungen</string>
-    <string name="pref_sesame_show_shortcuts_summary">Zeige Sesame shortcuts in Symbol Blasen</string>
+    <string name="pref_sesame_show_shortcuts_summary">Zeige Sesame-Einträge in App-Verknüpfungen</string>
     <string name="allow">Erlauben</string>
     <string name="sesame_enable_integration_title">Aktiviere Integration</string>
-    <string name="sesame_enable_integration_description"><![CDATA[Aktiviert das Teilen der Daten zwischen Lawnchair und Sesame. Du kannst das in den Lawnchair oder Sesame Einstellungen ändern.]]></string>
-    <string name="sesame_version">Integrationsversion</string>
-    <string name="sesame_shortcut_disabled">Sesame shortcuts nicht verfügbar</string>
-    <string name="pref_sesame_sync_icon_pack">Icon Pack teilen</string>
-    <string name="pref_sesame_sync_icon_pack_summary">Das Icon Pack von Lawnchair mit Sesame synchronisieren</string>
+    <string name="sesame_enable_integration_description"><![CDATA[Aktiviert das Teilen der Daten zwischen Lawnchair und Sesame. Dies kann in den jeweiligen Einstellungen geändert werden.]]></string>
```

Comment:
`<![CDATA[` and `]]>` generates problem with Gradle: https://github.com/LawnchairLauncher/Lawnchair/issues/1565